concrete-integer/src/server_key/crt.rs

```diff
@@ -250,8 +196,13 @@ impl ServerKey {
         F: Fn(u64) -> u64,
     {
         let basis = ct1.message_modulus_vec.clone();
-        for (i, block) in ct1.ct_vec.iter_mut().enumerate() {
-            let acc = self.key.generate_accumulator(|x| f(x) % basis[i]);
+
+        let accumulators = basis
```

Comment:
We should not need to clone here and instead directly do ct1.message_modulus_vec.iter().